securedrop_client/queue.py

```diff
@@ -41,10 +41,15 @@ def _process(self, exit_loop: bool) -> None:
                 job._do_call_api(self.api_client, session)
```

Comment:
shouldn't `self.last_job` get set back to None after this line so that we don't enter a forever retry the same job loop?

Reply:
oops nevermind I see that you're setting it to None here: https://github.com/freedomofpress/securedrop-client/pull/404/files/447a166a374ac0149ba45ec73de1c6fee17751fc#diff-dfebdbc5258e6805fadd416909ed2a90R36 (it got folded in my diff)